src/commands/ext-install.ts

```diff
@@ -194,6 +228,61 @@ interface InstallExtensionOptions {
   force?: boolean;
 }
 
+/**
+ * Saves the extension instance config values to the manifest.
+ *
+ * Requires running `firebase deploy` to install it to the Firebase project.
+ * @param options
+ */
+async function installToManifest(options: InstallExtensionOptions): Promise<void> {
+  const { projectId, extensionName, extVersion, paramsEnvPath, nonInteractive, force } = options;
+  const spec = extVersion?.spec;
+  if (!spec) {
+    throw new FirebaseError(
+      `Could not find the extension.yaml for ${extensionName}. Please make sure this is a valid extension and try again.`
+    );
+  }
+
+  const config = manifest.loadConfig(options);
+
+  let instanceId = spec.name;
```

Comment:
Yeah I did consider that.. another thing this code does is avoiding a 1/1679616 collision 😅 

we could start with a suffix if you think that's better UX?

Reply:
Ok now I remember another reason why I did this... before we were checking if the name matches an existing, we ask them if they want to update/reconfig instead. I added reasonings on why I don't think we should keep that in the PR description. But this means we need to prevent name duplicates all together

Reply:
Makes sense to me that we need to keep the .instanceExists() check. No need to include it here, but during UX review, lets ask if we should prompt for a name on the first instance too